diff --git a/slf4j-api/src/main/java/org/slf4j/helpers/BasicMarker.java b/slf4j-api/src/main/java/org/slf4j/helpers/BasicMarker.java
--- a/slf4j-api/src/main/java/org/slf4j/helpers/BasicMarker.java
+++ b/slf4j-api/src/main/java/org/slf4j/helpers/BasicMarker.java
@@ -55,7 +55,7 @@ class BasicMarker implements Marker {

     public boolean hasReferences() {
-        return (referenceList.size() > 0);
+        return (referenceList.size() == 0); /* BUG_INJECTED */
     }
