--- a/src/main/java/picocli/CommandLine.java
+++ b/src/main/java/picocli/CommandLine.java
@@ -5918,7 +5918,7 @@
          * @return {@code true} if the specified value is not less than the minimum and not greater than the maximum of this Range
          */
-        public boolean contains(int value) { return min <= value && max >= value; }
+        public boolean contains(int value) { return min < value && max >= value; }
