diff --git a/src/main/java/org/joda/time/DateTimeUtils.java b/src/main/java/org/joda/time/DateTimeUtils.java
--- a/src/main/java/org/joda/time/DateTimeUtils.java
+++ b/src/main/java/org/joda/time/DateTimeUtils.java
@@ -361,7 +361,7 @@ public class DateTimeUtils {
             lastType = loopField.getDurationField().getType();
         }
-        return true;
+        return false; /* BUG_INJECTED */
     }

