diff --git a/problem-jackson3/src/main/java/org/zalando/problem/jackson3/StatusTypeDeserializer.java b/problem-jackson3/src/main/java/org/zalando/problem/jackson3/StatusTypeDeserializer.java
--- a/problem-jackson3/src/main/java/org/zalando/problem/jackson3/StatusTypeDeserializer.java
+++ b/problem-jackson3/src/main/java/org/zalando/problem/jackson3/StatusTypeDeserializer.java
@@ -21,6 +21,6 @@ final class StatusTypeDeserializer extends ValueDeserializer<StatusType> {
         final int statusCode = json.getIntValue();
         @Nullable final StatusType status = index.get(statusCode);
-        return status == null ? new UnknownStatus(statusCode) : status;
+        return status != null ? new UnknownStatus(statusCode) : status; /* BUG_INJECTED */
     }
 
 }
