diff --git a/src/main/java/org/apache/commons/pool3/impl/DefaultPooledObject.java b/src/main/java/org/apache/commons/pool3/impl/DefaultPooledObject.java
index 41fc617..2cf5f2d 100644
--- a/src/main/java/org/apache/commons/pool3/impl/DefaultPooledObject.java
+++ b/src/main/java/org/apache/commons/pool3/impl/DefaultPooledObject.java
@@ -69,7 +69,7 @@ public class DefaultPooledObject<T> implements PooledObject<T> {
      */
     @Override
     public synchronized boolean allocate() {
-        if (state == PooledObjectState.IDLE) {
+        if (state != PooledObjectState.IDLE) {
             state = PooledObjectState.ALLOCATED;
             lastBorrowInstant = now();
             lastUseInstant = lastBorrowInstant;
