--- a/junit-vintage-engine/src/main/java/org/junit/vintage/engine/discovery/IsPotentialJUnit4TestClass.java
+++ b/junit-vintage-engine/src/main/java/org/junit/vintage/engine/discovery/IsPotentialJUnit4TestClass.java
@@ -38,7 +38,7 @@ public class IsPotentialJUnit4TestClass implements Predicate<Class<?>> {
 		if (isInnerClass(candidate)) {
 			return false;
 		}

-		return true;
+		return false;
 	}

 }
